doc/monitoring/grafana_dashboard.rst

```diff
@@ -71,14 +72,14 @@ metrics path as it was configured on Tarantool instances:
         metrics_path: "/metrics/prometheus"
 
 
-To collect metrics for InfluxDB, you must use Telegraf agent.
+To collect metrics for InfluxDB, use the Telegraf agent.
 First off, configure Tarantool metrics output in ``json`` format
 with :ref:`cartridge.roles.metrics <cartridge-role>` configuration or
-corresponding :ref:`output plugin <json>`. To start collecting metrics,
+corresponding :ref:`JSON output plugin <json>`. To start collecting metrics,
 add `http input <https://github.com/influxdata/telegraf/blob/release-1.17/plugins/inputs/http/README.md>`_
 to Telegraf configuration including each Tarantool instance metrics URL:
 
-..  code-block:: text
+..  code-block:: toml
```

Comment:
It really is, check https://raw.githubusercontent.com/influxdata/telegraf/release-1.17/plugins/inputs/http/README.md